frontend/src/questions/utils/QuestionQueries.js

```diff
@@ -12,5 +12,17 @@ export function getQuestionsLS() {
 }
 
 export function setQuestionsLS() {
-  return setLocalStorage(QUESTION_STORAGE_KEY, questionData);
+  const existingData = getLocalStorage(QUESTION_STORAGE_KEY);
+
+  // Check if local storage is empty for the specified key
+  if (!existingData) {
+    setLocalStorage(QUESTION_STORAGE_KEY, questionData);
+  }
```

Comment:
The issue might be here

Reply:
local storage is only needs to be set up once. But as of the current code, whenever we refresh or restart, the local storage is always being updated by the questionData.js, so it the questions will always reset to default state.